src/main/resources/liquibase/crop_changelog/20_3_0.xml

```diff
@@ -1,9 +1,11 @@
 <?xml version="1.1" encoding="UTF-8" standalone="no"?>
+
```

Comment:
@mchabito revert changes in this file

Reply:
@mchabito please revert this change, it was reverted but it was back in the last commit @rvtulod